test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -63,6 +63,14 @@ Describe 'Classes inheritance syntax' -Tags "CI" {
         [MyComparable].GetInterface("System.IComparable") | Should -Not -BeNullOrEmpty
     }
 
+    It 'can implement .NET interface properties' {
+        Add-Type -TypeDefinition 'public interface I1 { int I{ get; set; } }'
```

Comment:
Please replace "I" with more readable name. Maybe `InterfaceProperty`?

Reply:
Update both type names and the property name to be more descriptive